src/app/public/modules/text-editor/url-modal/text-editor-url-modal.component.html

```diff
@@ -0,0 +1,103 @@
+<sky-modal>
+  <sky-modal-header>
+    Create link
+  </sky-modal-header>
+  <sky-modal-content>
+    <sky-tabset
+      [active]="activeTab"
+      (activeChange)="activeTabChanged($event)"
+    >
+      <sky-tab
+        tabHeading="Web page"
+      >
+        <div class="sky-form-group sky-text-editor-url-modal-first-field">
+          <sky-input-box>
+            <label
+              class="sky-control-label sky-control-label-required"
+              [for]="urlInput.id"
+            >
+              Enter a URL to link to:
+            </label>
+            <input
+              class="sky-form-control"
+              type="text"
+              [(ngModel)]="url"
+              skyId
+              #urlInput="skyId"
+            />
+          </sky-input-box>
+        </div>
+        <sky-input-box class="sky-form-group">
+          <label
+            class="sky-control-label"
+            [for]="targetInput.id"
+          >
+            Where should it open?
+          </label>
+          <select
+            class="sky-form-control"
+            [(ngModel)]="target"
+            skyId
+            #targetInput="skyId"
+          >
+            <option value="0">Current window</option>
+            <option value="1">New window</option>
+          </select>
+        </sky-input-box>
+      </sky-tab>
+      <sky-tab
+        tabHeading="Email address"
+      >
+        <div class="sky-form-group first-field">
+          <sky-input-box>
+            <label
+              class="sky-control-label sky-control-label-required"
+              [for]="emailAddressInput.id"
+            >
+              Email address
+            </label>
+            <input
+              class="sky-form-control"
+              type="text"
+              [(ngModel)]="emailAddress"
```

Comment:
Seems like maybe we should be using reactive forms?

Reply:
Agreed. Decided to defer this for the beta release. Will create a bug to fix.